### PR TITLE
More developer documentation

### PR DESCRIPTION
**Pull request by lowjoel**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -8,3 +8,9 @@
 #
 # Remember to ensure that the commands in this file are idempotent!
 Instance.find_or_create_by(name: 'Default', host: '*')
+
+# Create the default user account.
+user = UserEmail.find_by_email('test@example.org')
+unless user
```
Comment (jsyeo) [inline db/seeds.rb]: NewbZ question: can we use `find_or_create_by`? It does the same thing as what you're testing for here isn't it? Or is there some gotchas?

Comment (lowjoel) [inline db/seeds.rb]: Yes and no... `email` is a virtual column, it doesn't exist in the User table.

Comment (jsyeo) [inline db/seeds.rb]: Oh. Ok.

Comment (jsyeo) [inline db/seeds.rb]: Oh right... There's `UserEmail` and `User`.

**Pull request merged**